pkg/claimhelper/claimhelper.go

```diff
@@ -374,3 +376,30 @@ func CreateClaimRoot() *claim.Root {
 		},
 	}
 }
+
+func SanitizeClaimFile(claimFileName, labelsFilter string) (string, error) {
+	log.Info("Sanitizing claim file %s", claimFileName)
+	data, err := ReadClaimFile(claimFileName)
+	if err != nil {
+		log.Error("ReadClaimFile failed with err: %v", err)
+		return "", err
+	}
+	var aRoot claim.Root
+	UnmarshalClaim(data, &aRoot)
+
+	// Remove the results that do not match the labels filter
+	for testID := range aRoot.Claim.Results {
+		if aRoot.Claim.Results[testID].TestID.Id != labelsFilter {
```

Comment:
Does this work for a `labelsFilter` that is not the name of a test case? For instance for a test suite name or a label such as `telco` or `all`.

Reply:
You could make it work for any `labelsFilter` by using the `labelsExprEvaluator` (and making it available outside the `checksdb` package) like this:

`if !labelsExprEvaluator.Eval(aRoot.Claim.Results[testID].TestID.Id) {
// delete test
}`

Otherwise I think that the functionality is too restricted to make a flag out of it.

Reply:
Done, this works.

Reply:
I couldn't make this work, looks like for each test ID read from the claim file, we need to generate metadata with identifiers.GetTestIDAndLabels(identifiers.TestSysAdminIdentifier), including the suite name and tag

Reply:
Ah I see, yes I wonder if I change:
`if !evaluator.Eval([]string{aRoot.Claim.Results[testID].TestID.Id})`
to 
`if !evaluator.Eval([]string{aRoot.Claim.Results[testID].TestID.Id, aRoot.Claim.Results[testID].TestID.Suite})`

Since the Eval() func accepts a slice of strings to evaluate.  I'll try that out.

Reply:
Okay yes this worked.  I was able to run:
`./certsuite run --sanitize-claim -l access-control`

And it keeps all of the access-control related tests in the claim output.

Reply:
What about just calling the function to return the labels to match on with something like:
```
		_,labels:=identifiers.GetTestIDAndLabels(*aRoot.Claim.Results[testID].TestID)
		if !evaluator.Eval( labels) {
```
That would also cover tags

Reply:
This works as well.  Updated!